src/lib.rs

```diff
@@ -120,3 +120,61 @@ extern crate self as com;
 // for code that uses `#![no_std]`.
 #[doc(hidden)]
 pub extern crate alloc;
+
```

Comment:
I think we should move this to a ref count module

Reply:
Done